### PR TITLE
Release/1.4.3

### PR DESCRIPTION
**Pull request by conradocanas**

### CODE REVIEW

File: .github/workflows/s3_buckets_build_and_upload_dev.yml [hunk 1]
```diff
@@ -54,4 +54,4 @@ jobs:
     # Uploading files to S3 bucket
     - name: Sync files to S3 bucket
       run: |
-        aws s3 sync ./.output/public s3://${{ secrets.S3_BUCKET }} --delete
+        aws s3 sync ./.output/public s3://${{ vars.S3_BUCKET }} --delete
```
Comment (toninorair) [inline .github/workflows/s3_buckets_build_and_upload_dev.yml:57]: what is the reason for this change?

File: cypress/e2e/emergencyProposals/proposals-emergency-batch-voting.cy.ts [hunk 2]
```diff
@@ -1,78 +0,0 @@
-it.skip("Proposals", () => {
```
Comment (toninorair) [inline cypress/e2e/emergencyProposals/proposals-emergency-batch-voting.cy.ts:1]: why is it deleted? it was skipped to start from

**Review by toninorair — APPROVED** [review]
Left 2 questions, change of secrets to vars looks somewhat suspicious

File: .github/workflows/s3_buckets_build_and_upload_dev.yml (hunk 1 above)
Comment (conradocanasm0) [reply .github/workflows/s3_buckets_build_and_upload_dev.yml:57]: this is a change in dev flow, already working

File: cypress/e2e/emergencyProposals/proposals-emergency-batch-voting.cy.ts (hunk 2 above)
Comment (conradocanasm0) [reply cypress/e2e/emergencyProposals/proposals-emergency-batch-voting.cy.ts:1]: It is a very old test, I would like to completely re-write it if needed in the future, more so if we are changing the layout of the whole application
